server/server.js

```diff
@@ -1,15 +1,36 @@
 var express = require('express');
+var expressLogging = require('express-logging');
+var logger = require('logops');
 var path = require('path');
-var gameHandler = require('./handlers/game_handler');
+var Cookies = require('universal-cookie');
+var firebaseRequests = require('./firebase.js');
+var gameHandler = require('./handlers/game_handler.js');
+
 var app = express();
 
 app.use("/static", express.static(path.join(__dirname, "/../public")))
 
+app.use(expressLogging(logger));
```

Comment:
what's that?

Reply:
server-side logging middleware (shows details about http requests  & responses to/from server)